From 4e36dc6840e962359bb7bc6ed94283af1c5598e3 Mon Sep 17 00:00:00 2001
From: Rob Browning <rlb@defaultvalue.org>
Date: Tue, 20 Feb 2018 23:58:09 -0600
Subject: Mark test-out-of-memory as an expected failure for now

Since this test has been failing for a long time on various
architectures, mark it as expected to fail for now so that we'll still
run it and can see the results, but won't be blocked by it.

There are known issues with the test upstream, and at least in some past
cases it's been possible to reproduce the failure quickly by running
test-suite/standalone/test-out-of-memory in a loop.

Bug: https://debbugs.gnu.org/29464
Bug-Debian: https://bugs.debian.org/880148
Bug-Debian: https://bugs.debian.org/966300
---
 test-suite/standalone/Makefile.am | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/test-suite/standalone/Makefile.am b/test-suite/standalone/Makefile.am
index e87100c96d..66bb3a5fb8 100644
--- a/test-suite/standalone/Makefile.am
+++ b/test-suite/standalone/Makefile.am
@@ -24,6 +24,7 @@ include $(top_srcdir)/am/snarf
 
 # initializations so we can use += below.
 TESTS =
+XFAIL_TESTS =
 noinst_LTLIBRARIES =
 check_PROGRAMS = 
 check_SCRIPTS =
@@ -299,6 +300,6 @@ check_SCRIPTS += test-stack-overflow
 TESTS += test-stack-overflow
 
 check_SCRIPTS += test-out-of-memory
-TESTS += test-out-of-memory
+XFAIL_TESTS += test-out-of-memory
 
 EXTRA_DIST += ${check_SCRIPTS}
